internal/handlers/images.go

```diff
@@ -14,11 +14,12 @@ import (
 const defaultArch = "x86_64"
 
 type ImageHandler struct {
-	long     http.Handler
-	byAPIKey http.Handler
-	byID     http.Handler
-	byToken  http.Handler
-	initrd   http.Handler
+	long           http.Handler
+	byAPIKey       http.Handler
+	byID           http.Handler
+	byToken        http.Handler
+	initrd         http.Handler
+	initrdAddrsize http.Handler
```

Comment:
Also here, let's rename to `s390xInitrdAddrsize`, etc

Reply:
Done